src/main/scala/de/hpi/epic/pricewars/utils/JSONConverter.scala

```diff
@@ -8,7 +8,7 @@ object JSONConverter extends SprayJsonSupport with DefaultJsonProtocol {
   implicit val shippingTimeFormat: RootJsonFormat[ShippingTime] = jsonFormat2(ShippingTime.apply)
   implicit val offerFormat: RootJsonFormat[Offer] = jsonFormat10(Offer.apply)
   implicit val offerPatchFormat: RootJsonFormat[OfferPatch] = jsonFormat9(OfferPatch)
-  implicit val merchantFormat: RootJsonFormat[Merchant] = jsonFormat5(Merchant.apply)
+  implicit val merchantFormat: RootJsonFormat[Merchant] = jsonFormat6(Merchant.apply)
```

Comment:
5,6,7? What's that?

Reply:
Wasn't implemented by me. It loads the defined class and the number corresponds to the attributes of each class. I'll add a comment